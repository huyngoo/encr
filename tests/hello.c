#include <stdio.h>

int main(int argc, char *argv[]){
	printf("Hello world, my name is Huy Ngo!\n");
	return 0;
}
