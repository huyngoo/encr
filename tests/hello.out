Hello world, my name is Huy Ngo!!!!!!!!!!
